tools/projmgr/test/data/TestSolution/PackLocking/pack_version_not_available.cbuild-pack.yml

```diff
@@ -0,0 +1,5 @@
+cbuild-pack:
+  resolved-packs:
+    - resolved-pack: ARM::RteTest_DFP@0.1.0
```

Comment:
How can a pack be resolved to a lower version than the minimum required?

Reply:
[Update cbuild-pack.yml](https://github.com/Open-CMSIS-Pack/devtools/pull/1788/commits/3c8bb4bee26bdd0ce1ed5c8f2c9852dd536a0d65)

Reply:
Yes, It was a mistake
[Update pack_version_not_available.csolution.yml](https://github.com/Open-CMSIS-Pack/devtools/pull/1788/commits/3d3cf419c87a2ccc5bcaf1a744d42010f03d22a0)